modules/core/src/views/orthographic-view.js

```diff
@@ -21,21 +30,15 @@ export default class OrthographicView extends View {
     // NOTE: automatically calculated from width and height if not provided
     const {
       // projection matrix arguments
-      left = -width / 2, // Left bound of the frustum
-      top = -height / 2, // Top bound of the frustum
+      left = (-width / 2 + offset[0]) * zoom, // Left bound of the frustum
```

Comment:
@ibgreen I wasn't sure about this part --- if users pass the left/top/right/bottom through props, then the zoom/pan interaction is gone.

Reply:
>  if users pass the left/top/right/bottom through props, then the zoom/pan interaction is gone.

It probably overrides the `viewState`. The design is that what is being shown should be defined by the `viewState`, the `View` should describe "how

Reply:
> Do we supposed to pass in views (array), viewStates (dictionary) and layerFilter (function) as props to <DeckGL ...> all at once?

Yes. Happy to hear suggestions for improvements. E.g. as discussed, it would be nice to support e.g. a `layerFilter` per view, rather than global on `Deck`, but that didn't make it into 6.1.

> @ibgreen @javidhsueh what is the suggested way of overriding the offset and zoom without accessing the controller? (if possible)

If you just want to override some of the props that the controller is giving you, you can use the new `View.viewState` prop in 6.1. If that is not what you need, I probably need to take a deeper look at the use case.